Tests/js/views/assets/ez-usermenuview-tests.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuview-tests', function (Y) {
+    var renderTest, eventTest,
```

Comment:
after adding the `addMenuItem` method, you'll also have to add a dedicated test case for it (like for any public method). A test is missing for the clickoutside event handler (you can add a method in the test about `displayed`).
You should also add a test for the valueFn of the `items` attribute. To avoid adding too many dependencies, in the setUp of this test you can write something like:
```js
Y.eZ.UserMenuItemFireEventView = Y.View;
```
and then you just have to check that the view in `items` receives the correct title and eventName and that the UserMenuView is an event target of the UserMenuItemFireEventView.